database/test_data.py

```diff
@@ -450,6 +450,16 @@
         "contactEmail": "",
         "contactName": ""
     },
+    {
+        "id": 9,
+        "programName": "CELTS-Sponsored Event",
+        "programUrl": "https://www.berea.edu/celts/",
```

Comment:
dead link, it's actually https://www.berea.edu/centers/center-for-excellence-in-learning-through-service

you're leading your own poor staff and students astray. thats bad and you should feel bad